community/contributing/ways_to_contribute.rst

```diff
@@ -75,10 +75,10 @@ Contributing code changes upstream has two big advantages:
    in some cases it might still be relevant to keep your changes only for your
    own project, if they are too specific.
 
--  The whole community will benefit from your work, and other contributors will
-   behave the same way, contributing code that will be beneficial to you. At
-   the time of this writing, more than 1000 developers have contributed code
-   changes to the engine!
+-  The whole community will benefit from your work and similarly you will be 
+   benefited by the code contributed by other contributors all over the world. 
```

Comment:
```suggestion
-  The whole community will benefit from your work and similarly you will 
   benefit from the code contributed by other contributors all over the world. 
```

In English you say "you will benefit" rather than "you will be benefited"